Sources/PodToBUILD/ObjcLibrary.swift

```diff
@@ -388,14 +381,22 @@ public struct ObjcLibrary: BazelTarget, UserConfigurable, SourceExcludable {
         let extraDepNames = extraDeps.map { bazelLabel(fromString: ":\($0)") }
         deps = AttrSet(basic: extraDepNames) <> mpPodSpecDeps
 
+        // Adds minimal, non specified Xcode defaults
         let extraCopts: AttrSet<[String]>
         if case .cpp = sourceType {
```

Comment:
It's too bad Swift's `case`'s aren't expressions.

Reply:
I think they might have added this in Swift 5. 🥳